aws/data_source_aws_ec2_managed_prefix_list.go

```diff
@@ -0,0 +1,153 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
```

Comment:
This should fix the `importlint` error:

```suggestion
	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
```